backend/routes/user.routes.ts

```diff
@@ -21,5 +22,6 @@ router.post('/loginStudent', loginStudent);
 router.post('/logoutStudent', verifyAuthentication, logoutStudent);
 router.post('/resetPassword', resetPassword);
 router.get('/getAllStudents', getAllStudents);
+router.patch('/updateUser/:id', updateUser);
```

Comment:
this needs to have an auth middleware assigned to it, since it is a protected route. See line 22 for this ```router.post('/logoutStudent', verifyAuthentication, logoutStudent);```